src/modules/CanHelp/Outlet.vue

```diff
@@ -155,7 +155,7 @@ export default class CanHelpOutlet extends Vue {
   filteredOutlets = [] as ResOutlet[];
   towns = ["Wszystkie"];
   selectedTown = "";
-  path = "/api/requests/";
+  path = "http://localhost:5011/help-request";
```

Comment:
Again, we cannot merge it, as this breaks current compatibility, and we shouldn't really use hardcoded ports.
I think the problem lies within our routing. Is nest's API served under `/api` ?

Reply:
yes let's try /api/